Documentation/architecture/cross-platform-cryptography.md

```diff
@@ -48,6 +42,44 @@ This would result in a `PlatformNotSupportedException` when invoking the `Create
 In the future there is a possibility that new cipher/chaining modes may be added to .NET Core and that these new modes may not apply to all symmetric algorithms.
 This would likely result in a `NotSupportedException` when using the set-accessor of the `Mode` property on the `SymmetricAlgorithm` object, but this prediction is subject to change.
 
+## Authenticated Encryption
+
+Authenticated Encryption (AE) support is provided for AES-CCM and AES-GCM via `System.Security.Cryptography.AesCcm` and `System.Security.Cryptography.AesGcm`, respectively.
+
+On macOS the AesCcm and AesGcm classes use OpenSSL for support.
```

Comment:
Is this because macOS's built-in crypto support doesn't provide these algorithms?  Might be worth mentioning.